maptivist/src/main/webapp/index.html

```diff
@@ -102,9 +102,9 @@ <h1>Fill in Marker</h1>
 
   </body>
 
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+	<!-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
 	<script async defer src="https://apis.google.com/js/api.js"
         onload="this.onload=function(){};handleClientLoad();closeForm()"
         onreadystatechange="if (this.readyState === 'complete') this.onload()">
-	</script>
+	</script> -->
```

Comment:
It's best practice just to delete code instead of commenting it out, unless you expect that the code will be used again soon. If you plan to use it again soon, then add a comment why it's being commented out and under what conditions it should be re-enabled.

That way others can re-enable if it necessary.